src/main/java/uk/gov/hmcts/reform/juddata/camel/processor/ICustomValidationProcessor.java

```diff
@@ -41,36 +41,36 @@ default void filterInvalidUserProfileRecords(List<T> filteredChildren,
                                                  JsrValidatorInitializer<T> jsrValidatorInitializer,
                                                  Exchange exchange, String logComponentName) {
         Type mySuperclass = getType();
-        List<String> invalidElinks = new ArrayList<>();
+        List<String> invalidPers = new ArrayList<>();
         if (nonNull(invalidJudicialUserProfileRecords)) {
 
             invalidJudicialUserProfileRecords.forEach(invalidRecords -> {
-                //Remove invalid appointment for user profile and add to invalidElinks List
+                //Remove invalid appointment for user profile and add to invalidPers List
                 boolean filteredRecord = false;
                 if (((Class) mySuperclass).getCanonicalName().equals(JudicialOfficeAppointment
                     .class.getCanonicalName())) {
                     filteredRecord = filteredChildren.removeIf(filterInvalidUserProfAppointment ->
-                        ((JudicialOfficeAppointment) filterInvalidUserProfAppointment).getElinksId()
-                            .equalsIgnoreCase(invalidRecords.getElinksId()));
+                        ((JudicialOfficeAppointment) filterInvalidUserProfAppointment).getPerId()
+                            .equalsIgnoreCase(invalidRecords.getPerId()));
                 } else if (((Class) mySuperclass).getCanonicalName().equals(JudicialOfficeAuthorisation
                     .class.getCanonicalName())) {
                     filteredRecord = filteredChildren.removeIf(filterInvalidUserProfAuthorization ->
-                        ((JudicialOfficeAuthorisation) filterInvalidUserProfAuthorization).getElinksId()
-                            .equalsIgnoreCase(invalidRecords.getElinksId()));
+                        ((JudicialOfficeAuthorisation) filterInvalidUserProfAuthorization).getPerId()
+                            .equalsIgnoreCase(invalidRecords.getPerId()));
                 }
                 if (filteredRecord) {
-                    invalidElinks.add(invalidRecords.getElinksId());
+                    invalidPers.add(invalidRecords.getPerId());
                 }
             });
 
-            if (isNotEmpty(invalidElinks)) {
+            if (isNotEmpty(invalidPers)) {
                 //Auditing JSR skipped rows of user profile for Appointment/Authorization
```

Comment:
make as invalidPers as invalid PerId's trivial but lets be consitent

Reply:
Hi @abhijitdiwan , this change has been implemented.